project.py

```diff
@@ -81,13 +105,60 @@ def find_board(self, image):
 
         return our_cnt
 
+    def seed_filter(self, image, seed):
+        """
+        My ghetto green filter that only checks if we really hit the board.
+        So it checks if the green value of seed pixel is the highest
+
+        Args:
+            image(numpy.ndarra): Image from witch we get seed values
+            seed(tuple): Seed point
+
+        Returns:
+            bool: True if found the board. False otherwise
+        """
+        rgb = image[self.seed[1], self.seed[0]]
+        if rgb[1] > rgb[0] and rgb[1] > rgb[2]:
+            return True
+        return False
+
+    def get_seed(self, image):
+        """
+        Tries to find good seed for flood fill algorithm by randomly picking
+        a point somewhere in the middle of a image and then checking if its
+        value is correct
+
+        Args:
+            image(numpy.ndarray): Image to search in
+
+        Returns:
+            Coordinates of seed position
+        """
+        if self.seed is not None and self.seed_filter(image, self.seed):
+            return self.seed
+
+        h, w = image.shape[:2]
+        # three retries to find proper seed
+        for i in xrange(1, 3):
+            self.seed = (randint(w/4, (w/4)*3), randint(h/4, (h/4)*3))
```

Comment:
@mrshu I don't think it is necessary. Docstring already contains information about that `"...randomly picking a point somewhere in the middle of a image..."`.